src/plugins/dataRefs/dataRefs.ts

```diff
@@ -41,15 +56,67 @@ export function integrate(timestamps: number[], values: DataSeriesExtended) {
     return integral;
 }
 
+function getLast(values: DataSeriesExtended): number | null {
+    for (let i = values.length - 1; i >= 0; i--) {
+        const val = values[i];
+        if (val !== null && typeof val === 'number') {
+            return val;
+        }
+    }
+    return null;
+}
```

Comment:
Is it wrong to go through array backward? The first non-null value is our desired value.

Reply:
It's the function for `last` item in series.